OpenQA/Isotovideo/Runner.pm

```diff
@@ -244,6 +244,13 @@ sub _init_bmwqemu ($, @args) {
     }
 }
 
+sub init ($self, @args) {
+    $self->_init_bmwqemu(@ARGV);
+    $self->prepare;
+    $self->start_autotest;
+    $self->create_backend;
+}
+
```

Comment:
I'd rather just add that to `new` so we don't need to call `new` + `init` separately.

Reply:
ok I tried but failed. Because then at least t/19-isotovideo-command-processing.t fails as it wants to instantiate a clean runner object and not initialize everything. Also "new" is the constructor and IMHO shouldn't do that much if any dynamic evaluation but "init bmwqemu" and "start autotest" is already doing a lot. With that I prefer the separate `init`. WDYT?